### PR TITLE
Add bulk_remove_subscriptions support

### PR DESCRIPTION
**Pull request by swadeley**
Hello

this is used here: https://github.com/SatelliteQE/robottelo/pull/7982

It implements:
https://theforeman.org/plugins/katello/3.15/api/apidoc/v2/hosts_bulk_actions.html#description-remove_subscriptions

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -4286,6 +4303,11 @@ def path(self, which=None):
                 super(Host, self).path(which='base'),
                 which
             )
+        elif which in ('bulk/remove_subscriptions',):
+            return '{0}/{1}'.format(
+                super(Host, self).path(which='base'),
+                which
+            )
```
Comment (mirekdlugosz) [inline nailgun/entities.py:4310]: I feel this entire section from line 4296 should be in single branch:
```python
elif which in ('bulk/install_content', 'bulk/add_subscriptions', 'bulk/remove_subscriptions'):
            return '{0}/{1}'.format(
                super(Host, self).path(which='base'),
                which
            )
```

**Review by mirekdlugosz — APPROVED** [review]
ACK with suggestion

File: nailgun/entities.py (hunk 1 above)
Comment (swadeley) [reply nailgun/entities.py]: OK, looks neat

changes made
thank you

**Review by mirekdlugosz — DISMISSED** [review]
flake complains about one line being too long. please fix.

**Comment by swadeley** [comment]
> flake complains about one line being too long. please fix.

OK, fixed docstring

**Review by mirekdlugosz — APPROVED** [review]

**Review by pondrejk — APPROVED** [review]

**Comment by swadeley** [comment]
Thank you

**Pull request merged**